web/wp-content/themes/dctx-v2/template-parts/scaffolding/scaffolding-elements.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-38ennv.php, but found phpcs-scan-38enNV.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).